libsrc/apple2/crt0.s

```diff
@@ -76,6 +76,27 @@ init:   ldx     #zpspace-1
         ldx     #>$BF00
         bne     :+              ; Branch always
 
+set_lc_vectors:
```

Comment:
I know that this is really silly, but I'd like to at least mention that so far the style was to have code labels always on the line of the first actual instruction - and find names that are short enough to do so.